internal/config/config.go

```diff
@@ -661,6 +659,27 @@ func bgpAdvertisementsFromLegacyCR(ads []metallbv1beta1.LegacyBgpAdvertisement,
 	return ret, nil
 }
 
+func (ad *BGPAdvertisement) communtiesForBGPAd(adCommunities []string, communities map[string]uint32) error {
+	for _, c := range adCommunities {
+		v, err := ParseCommunity(c)
+		if err != nil {
+			if strings.Contains(err.Error(), "invalid community string") {
```

Comment:
Let's use an instance of error and errors.Is

Reply:
done